elliptic-curve/src/lib.rs

```diff
@@ -94,17 +94,18 @@ pub mod ecdh;
 #[cfg(feature = "jwk")]
 mod jwk;
 
-/// Optimized simplified Shallue-van de Woestijne-Ulas methods
-#[cfg(feature = "osswu")]
-#[cfg_attr(docsrs, doc(cfg(feature = "osswu")))]
-pub mod osswu;
-
 /// Traits for computing hash to field as described in
 /// <https://datatracker.ietf.org/doc/draft-irtf-cfrg-hash-to-curve>
-#[cfg(feature = "digest")]
-#[cfg_attr(docsrs, doc(cfg(feature = "digest")))]
+#[cfg(feature = "hash2curve")]
+#[cfg_attr(docsrs, doc(cfg(feature = "hash2curve")))]
 pub mod hash2field;
 
+/// Traits for computing hash to curve as described in
+/// <https://datatracker.ietf.org/doc/draft-irtf-cfrg-hash-to-curve>
+#[cfg(feature = "hash2curve")]
+#[cfg_attr(docsrs, doc(cfg(feature = "hash2curve")))]
+pub mod hash2curve;
```

Comment:
Here's another place it'd be nice to use toplevel module comments rather than comments on the respective modules.

Additionally I'd suggest splitting the links onto separate lines and keeping the first line short and sweet so it renders well in the rustdoc.

Here's what it looks like at present:

<img width="994" alt="Screen Shot 2022-01-06 at 2 37 59 PM" src="https://user-images.githubusercontent.com/797/148455767-870c654b-372c-498b-915a-425b43ba6cdc.png">

Keeping the descriptions short will prevent the `hash2curve` feature label from linewrapping.